ci-operator/step-registry/cucushift/upgrade/paused/cucushift-upgrade-paused-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-upgrade-paused
+  steps:
+    - ref: idp-htpasswd
+    - ref: cucushift-pre
+    - ref: cucushift-upgrade-prepare
+    - ref: cucushift-upgrade-paused-intermediate
+    - ref: cucushift-upgrade-paused-intermediate-sanity
+    - ref: cucushift-upgrade-paused-latest
```

Comment:
For nightly build, this ref is not suitable which use `oc adm upgrade --to-latest` as upgrade command.

Reply:
Would you mind to log an jira issue and show the correct steps in the jira issue ?

Reply:
sure, I could give the exact eus ci steps in the jira ticket, reviewing what we have currently and what we can re-use here.

Reply:
Looks like cucushift-upgrade-paused-latest step is using `oc adm upgrade --to-image` [cli](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/paused/latest/cucushift-upgrade-paused-latest-commands.sh#L9) which looks good.

Reply:
ah, you're right, the `latest` keyword make me confused with `cucushift-upgrade-paused-intermediate-to-latest`